evalml/tests/objective_tests/test_standard_metrics.py

```diff
@@ -437,6 +438,23 @@ def test_mcc_catches_warnings():
         assert len(record) == 0
 
 
+def test_mape_time_series_model():
+    obj = MAPE()
+
+    s1_actual = np.array([0, 0, 1, 1, 1, 1, 2, 0, 2])
+    s1_predicted = np.array([0, 1, 0, 1, 1, 2, 1, 2, 0])
+
+    s2_actual = np.array([1, 2, 3, 2, 1, 2, 1, 0])
+    s2_predicted = np.array([0, 2, 2, 1, 3, 2, 0, 1])
+
+    s3_actual = np.array([1.5, 2.5, -1.5, 0, 1, 3, 2.4, -3.2])
+    s3_predicted = np.array([1.5, 1.5, -1, 1, 1.1, -3, 2, -3])
+
+    assert obj.score(s1_actual, s1_predicted) == pytest.approx(3.5 / 6 * 100)
+    assert obj.score(s2_actual, s2_predicted) == pytest.approx(4.83333333 / 7 * 100)
+    assert obj.score(s3_actual, s3_predicted) == pytest.approx(3.0625 / 7 * 100)
```

Comment:
@ParthivNaresh nice 👍  I was able to walk through by hand and reproduce the math for the first example you have here. Only thought here is that if you could simplify the numbers a bit, make the arrays shorter,  make some of those decimals more round etc., it would help make this math easier to follow.